data/sql/updates/pending_db_world/rev_1621339467730285597.sql

```diff
@@ -0,0 +1,8 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1621339467730285597');
+
+DELETE FROM `spell_dbc` WHERE `ID` = 1543;
```

Comment:
please remove the `DELETE` in case of `spell_dbc` (or other DBC tables) we use only `INSERT IGNORE`

Reply:
Alrighty, done